tokio/src/sync/notify.rs

```diff
@@ -275,13 +287,41 @@ impl AtomicNotification {
     /// Store-release a notification.
     /// This method should be called exactly once.
     fn store_release(&self, notification: Notification) {
-        self.0.store(notification as usize, Release);
+        let data: usize = match notification {
+            Notification::All => NOTIFICATION_ALL & NOTIFICATION_TYPE_MASK,
+            Notification::One(NotifyOneStrategy::Fifo) => {
+                (((NOTIFICATION_NOTIFY_ONE_STRATEGY_FIFO)
+                    << NOTIFICATION_NOTIFY_ONE_STRATEGY_SHIFT)
+                    & NOTIFICATION_NOTIFY_ONE_STRATEGY_MASK)
+                    | (NOTIFICATION_ONE & NOTIFICATION_TYPE_MASK)
+            }
+            Notification::One(NotifyOneStrategy::Lifo) => {
+                (((NOTIFICATION_NOTIFY_ONE_STRATEGY_LIFO)
+                    << NOTIFICATION_NOTIFY_ONE_STRATEGY_SHIFT)
+                    & NOTIFICATION_NOTIFY_ONE_STRATEGY_MASK)
+                    | (NOTIFICATION_ONE & NOTIFICATION_TYPE_MASK)
```

Comment:
This is pretty hard to read. Can we just make three constants equal to `0b00`, `0b01`, `0b10` or whatever the values are?

Reply:
Good call, Ive changed for using just constants which makes definitely more readable the code